helpers/build.go

```diff
@@ -0,0 +1,3 @@
+package helpers
+
+const CreateSnapshotAnnotationName = "test.appstudio.openshift.io/succeed-create-snapshot"
```

Comment:
```suggestion
const CreateSnapshotAnnotationName = "test.appstudio.openshift.io/create-snapshot-status"
```

`suceeed` sounds confusing given it can also report error

Reply:
If I'm not wrong, I'm still seeing `test.appstudio.openshift.io/succeed-create-snapshot` on this line